data/json/items/comestibles/carnivore.json

```diff
@@ -124,6 +124,20 @@
     }
   },
   {
+    "id": "offal_cooked",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": "cooked offal",
+    "name_plural": "cooked offal",
+    "description": "Freshly cooked entrails.  Unappetizing, but filled with essential vitamins.",
+    "relative": {
+        "vitamins": [ [ "vitA", 5 ] ]
```

Comment:
Yeah, copying from offal is a better idea here. If offal gets some significant change, such as vitamins, nutrition, health etc., cooked offal would be adjusted automatically. Say, addition of vitamin D or splitting offal into liver and other offal.